assets/javascripts/redmine_last_seen_issues.js

```diff
@@ -0,0 +1,52 @@
+// global: positioning last_seen_issues entry before logged in div
+$(document).ready(function () {
+    var $login_menu = $("div#loggedas");
+    var $li = $(".last-seen-issues");
+    $($li).prependTo($login_menu);
```

Comment:
Isn't sufficient to say `$li.prependTo($login_menu)`?  Do you really need `$($li)`?

Reply:
Done.